45/mhered/app/fizzbuzz.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+
+
+def fizzbuzz(num: int):
+    """
+    FizzBuzz(int) plays FizzBuzz
+    """
+    if not isinstance(num, int):
```

Comment:
mypy would also catch this, but it does not prevent the function from being used by other types, after a `raise` you don't need the else so you could dedent the code below one level